rsmtool/model.py

```diff
@@ -571,6 +572,32 @@ def train_builtin_model(model_name, df_train, experiment_id, csvdir, figdir):
         # we used only the non-zero features
         used_features = non_zero_features
 
+    elif model_name == 'WeightedLeastSquares':
+
+        # train weighted least squares regression 
+        # get the feature columns
+
+        X = df_train[feature_columns]
+
+        # add the intercept
+        X = sm.add_constant(X)
+
+        # define the weights as inverse proportion of total number of data points for each score
+
+        score_levels = df_train['sc1'].unique()
+        expected_proportion = 1/len(score_levels)
+        score_weights_dict = dict([(sc1, expected_proportion/len(df_train[df_train['sc1']==sc1])) for sc1 in score_levels])
```

Comment:
Just realized, this could be simplified significantly if you just used `value_counts()` instead of `unique()` right?